pkgs/development/beam-modules/erlang-ls/default.nix

```diff
@@ -16,6 +16,12 @@ let
           substituteInPlace rebar.config --replace ", warnings_as_errors" ""
           '';
       });
+      json_polyfill = super.json_polyfill.overrideAttrs (_: {
+        # don't attempt to remove beam file during build, as it won't exist for erlang >= 27
```

Comment:
Could be worth explaining why it won't exist?

Reply:
@chiroptical I investigated and found out that the `beam` files aren't placed in `$REBAR_DEPS_DIR/json_polyfill/ebin` during `nix build`, but in `ebin/`, so I adjusted the path so the delete succeeds instead of removing it.

Updated the comment and `substituteInPlace`.